### PR TITLE
chore: Added meta tags for Search Engine Optimization.

### PR DESCRIPTION
**Pull request by Bearcin46**
I have added meta tags to the Tamil Calender App improve the Search Engine Optimization.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,6 +2,22 @@
...
+    />
+    <meta name="author" content="Anbu Selvan" />
+    <meta property="og:title" content="Calender App to display tamil days" />
+    <meta
+      property="og:description"
+      content="You can get all tamil days from 2023 to 2024. Just try it!"
+    />
+    <meta
+      property="og:image"
+      content="https://img.freepik.com/free-photo/purple-calendar-clock-search-icon-reminder-notification-concept-website-ui-purple-background-3d-rendering-illustration_56104-1318.jpg?w=826&t=st=1704259592~exp=1704260192~hmac=07c6595f4faa079beaa34a97937cc2b4fb9f3b8d9141f6e76ae640c04d43a776"
```
Comment (anburocky3) [inline index.html]: remove unnecessary url info after `.jpg`

File: index.html [hunk 2]
```diff
@@ -2,6 +2,22 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
+    <meta name="description" content="Tamil-Day-Calender" />
```
Comment (anburocky3) [inline index.html]: Description should be 155+ characters in length

**Review by anburocky3 — CHANGES_REQUESTED** [review]
Fix it for merging.

File: index.html (hunk 1 above)
Comment (Bearcin46) [reply index.html]: Resolved. Please check!

**Review by anburocky3 — APPROVED** [review]
Everything looks cool!

**Comment by anburocky3** [comment]
Thanks @Bearcin46

**Pull request merged**